lib/qdrant-docker-image-ecr-deployment-cdk-stack.ts

```diff
@@ -1,16 +1,26 @@
 import * as cdk from 'aws-cdk-lib';
 import { Construct } from 'constructs';
-// import * as sqs from 'aws-cdk-lib/aws-sqs';
+import * as ecrDeploy from 'cdk-ecr-deployment';
+import * as ecr from 'aws-cdk-lib/aws-ecr';
+import { QdrantDockerImageEcrDeploymentCdkStackProps } from './QdrantDockerImageEcrDeploymentCdkStackProps';
 
 export class QdrantDockerImageEcrDeploymentCdkStack extends cdk.Stack {
-  constructor(scope: Construct, id: string, props?: cdk.StackProps) {
-    super(scope, id, props);
+    constructor(scope: Construct, id: string, props: QdrantDockerImageEcrDeploymentCdkStackProps) {
+        super(scope, id, props);
 
-    // The code that defines your stack goes here
+        const ecrRepository = new ecr.Repository(this, `${props.appName}-QdrantDockerImageEcrRepository`, {
+            repositoryName: props?.repositoryName ?? 'qdrant-docker-image-ecr-deployment-cdk',
```

Comment:
**Suggestion:** The `repositoryName` property in the `ecr.Repository` constructor should not have a fallback since it is already defined as a required property in `QdrantDockerImageEcrDeploymentCdkStackProps`.
```suggestion
        const ecrRepository = new ecr.Repository(this, `${props.appName}-QdrantDockerImageEcrRepository`, {
            repositoryName: props.repositoryName,
            ...
        });
```